catalog.html

```diff
@@ -0,0 +1,277 @@
+            <span class="visually-hidden">Поиск.</span>
+          </a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">
+            <span class="visually-hidden">
+              Избранное.
+            </span>
+          </a>
+        </li>
+        <li class="navigation-item">
+          <button class="header-button" type="button">Хочу сюда</button>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="main-inner">
+    <div>
+      <header class="inner-header">
+        <h1 class="inner-header-title">
+          Гостиницы Седоны
+        </h1>
+        <ul class="breadcrumbs">
+          <li class="breadcrumbs-item">
+            <a class="breadcrumbs-link" href="index.html">
+              <span class="visually-hidden">Перейти на главную страницу.</span>
+            </a>
+          </li>
+          <li class="breadcrumbs-item">Гостиницы</li>
+        </ul>
+      </header>
+      <section class="catalog-hotels">
+        <h2 class="visually-hidden">
+          Список гостиниц с фильтрами.
+        </h2>
+        <form class="catalog-filter" action="https://echo.htmlacademy.ru/" method="get">
+          <fieldset class="catalog-filter-group">
+            <legend class="catalog-filter-title">Инфраструктура:</legend>
+            <label class="control">
+              <input class="control-input" type="checkbox" name="swimming pool" checked>
+              Бассейн
+            </label>
+            <label class="control">
+              <input class="control-input" type="checkbox" name="parking" checked>
+              Парковка
+            </label>
+            <label class="control">
+              <input class="control-input" type="checkbox" name="wi-fi">
+              Wi-Fi
+            </label>
+          </fieldset>
+          <fieldset class="catalog-filter-group">
+            <legend class="catalog-filter-title">Тип жилья:</legend>
+            <label class="control">
+              <input class="control-input" type="radio" name="type-housing" checked>
+              Гостиница
+            </label>
+            <label class="control">
+              <input class="control-input" type="radio" name="type-housing">
+              Мотель
+            </label>
+            <label class="control">
+              <input class="control-input" type="radio" name="type-housing">
+              Аппартаменты
+            </label>
+          </fieldset>
+          <fieldset class="catalog-filter-group">
+            <legend class="catalog-filter-title">Стоимость,₽</legend>
+          </fieldset>
+          <button class="button" type="submit">
+            Применить
+          </button>
+          <button class="button" type="reset">
+            Сбросить
+          </button>
+        </form>
+      </section>
+    </div>
+    <section>
+      <h3>
+        НАЙДЕНО ГОСТИНИЦ:38
+      </h3>
+      <select class="select-control" aria-label="Сортировка">
```

Comment:
```suggestion
      <select class="select-control" aria-label="Отсортировать гостиницы как:">
```

да, можно использовать aria-label вместо лейбла, единственный минус - он не переводится автопереводчиками, а так норм :) Отлично придумал!